src/components/List/List.tsx

```diff
@@ -201,11 +210,6 @@ export class List<T = unknown> extends React.Component<ListProps<T>, ListState<T
                 }
                 break;
             }
-            default: {
```

Comment:
These are breaking changes because after them the behavior that users expect will change